components/nav/common.js

```diff
@@ -55,15 +55,7 @@ export function Back () {
   if (!back) return null
 
   return (
-    <a
-      role='button' tabIndex='0' className='nav-link p-0 me-2' onClick={() => {
-        if (back) {
-          router.back()
-        } else {
-          router.push('/')
-        }
-      }}
-    >
```

Comment:
As there's a check on line 55 for "truthy" 'back' value then no need to check again here